omnigibson/robots/vx300s.py

```diff
@@ -0,0 +1,244 @@
+                corresponds to all modalities being used.
+                Otherwise, valid options should be part of omnigibson.sensors.ALL_SENSOR_MODALITIES.
+                Note: If @sensor_config explicitly specifies `modalities` for a given sensor class, it will
+                    override any values specified from @obs_modalities!
+            proprio_obs (str or list of str): proprioception observation key(s) to use for generating proprioceptive
+                observations. If str, should be exactly "default" -- this results in the default proprioception
+                observations being used, as defined by self.default_proprio_obs. See self._get_proprioception_dict
+                for valid key choices
+            sensor_config (None or dict): nested dictionary mapping sensor class name(s) to specific sensor
+                configurations for this object. This will override any default values specified by this class.
+            grasping_mode (str): One of {"physical", "assisted", "sticky"}.
+                If "physical", no assistive grasping will be applied (relies on contact friction + finger force).
+                If "assisted", will magnetize any object touching and within the gripper's fingers.
+                If "sticky", will magnetize any object touching the gripper's fingers.
+            kwargs (dict): Additional keyword arguments that are used for other super() calls from subclasses, allowing
+                for flexible compositions of various object subclasses (e.g.: Robot is USDObject + ControllableObject).
+        """
+        # Run super init
+        super().__init__(
+            prim_path=prim_path,
+            name=name,
+            class_id=class_id,
+            uuid=uuid,
+            scale=scale,
+            visible=visible,
+            fixed_base=fixed_base,
+            visual_only=visual_only,
+            self_collisions=self_collisions,
+            load_config=load_config,
+            abilities=abilities,
+            control_freq=control_freq,
+            controller_config=controller_config,
+            action_type=action_type,
+            action_normalize=action_normalize,
+            reset_joint_pos=reset_joint_pos,
+            obs_modalities=obs_modalities,
+            proprio_obs=proprio_obs,
+            sensor_config=sensor_config,
+            grasping_mode=grasping_mode,
+            **kwargs,
+        )
+
+    @property
+    def model_name(self):
+        return "VX300S"
+
+    @property
+    def discrete_action_list(self):
+        # Not supported for this robot
+        raise NotImplementedError()
+
+    def _create_discrete_action_space(self):
+        # Fetch does not support discrete actions
+        raise ValueError("VX300S does not support discrete actions!")
+
+    def update_controller_mode(self):
+        super().update_controller_mode()
+        # overwrite joint params (e.g. damping, stiffness, max_effort) here
```

Comment:
Maybe remove this function is there is no overwriting?

Reply:
done!